src/Client/Adapter/Exception/ConnectTimeoutException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Http\Client\Adapter\Exception;
+
+/**
+ */
```

Comment:
Please add a description detailing why this is different from other timeout exceptions